easybuild/easyconfigs/c/CPPE/CPPE-0.3.3a0-GCC-12.3.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'CMakeMake'
+
+name = 'CPPE'
+version = '0.3.3a0'
+
+homepage = 'https://github.com/maxscheurer/cppe'
+description = """CPPE is an open-source, light-weight C++ and Python library for Polarizable
+Embedding (PE)1,2 calculations. It provides an easy-to-use API to implement PE
+for ground-state self-consistent field (SCF) calculations and post-SCF methods.
+A convenient Python interface is also available."""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/maxscheurer/cppe/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['6dc3f863b5b7ffd1c049bf06e5d148bfdd08fc1c1aa2f5a506f211022628e2c3']
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('pybind11', '2.11.1'),
```

Comment:
This may need to be a runtime dependency, see also #21564 (I need to verify whether that actually makes sense)